Delete virtual machines on Libvirt

hashnao opened the pull request:
I want to add a playbook and an instruction to remove the VMs on Libvirt to re-create as it seems there's no related tasks nor instructions.

hashnao committed:
Delete virtual machines on Libvirt

noseka1 commented on delete_libvirt_vms.yml:
```diff
@@ -0,0 +1,6 @@
+- hosts: kvm
```
Why not use the existing "helper" host?

hashnao replied:
I just prefer to directly log in to the KVM host from the client by specifying the host in the master playbook rather than specifying the Libvirt connection URI as follows. One of the reasons is that it's easy to follow and the other is the root password is required several times when the Libvirt connection is used. Anyway, I believe documentation is enough to understand, but not committing the change indeed.

```
libvirt:
 qemu+ssh://root@192.168.150.1/system
```

hashnao replied:
I had a bit of misunderstanding and I found that the KVM host isn't necessarily set int the inventory file. See https://github.com/noseka1/openshift-auto-upi/pull/22#discussion_r493195990

noseka1 commented on delete_libvirt_vms.yml:
```diff
@@ -0,0 +1,6 @@
+- hosts: kvm
```
Can we rename the playbook to "delete_libvirt.yml"? It's a detail, we are not deleting the vms only but also the volumes.

noseka1 commented on docs/openshift_libvirt.md:
````diff
@@ -63,3 +63,14 @@ $ virsh net-start openshift
 ```
 $ virsh net-autostart openshift
 ```
+
````
Thanks for updating the docs as well! Can we move this paragraph to the front page, just after the "Adding Cluster Nodes" section? And probably change the title to "Deleting OpenShift" so it matches with the existing section "Installing OpenShift"?

hashnao replied:
Moved the paragraph to the front page.

noseka1 commented on inventory/helper.ini:
```diff
@@ -1,2 +1,3 @@
 [all]
 helper ansible_connection=local
+kvm ansible_host=KVM_HOSTNAME
```
I am not sure if I understand why the kvm host is introduced here. I think if somebody wants to use a remote host, they could set ansible_host=x.x.x.x ansible_connection=ssh ? Would that not work? Or perhaps I don't understand the purpose here.

hashnao replied:
Yes, that's correct. I don't have to set the kvm line in the inventory file even if I want to connect to the KVM host as `ansible_host=x.x.x.x ansible_connection=ssh` could be set with `-e` flag. I've removed the line and added a few comments in the README file instead.

noseka1 reviewed (commented):
@hashnao Thanks a lot for the great work! This feature was really missing. The users will be able to quickly delete the libvirt deployment and start over.

hashnao committed:
Rename delete_libvirt_vms.yml to delete_libvirt.yml

hashnao committed:
Update README.md

Removing the Deleting VMs on Libvirt paragraph to the front page https://github.com/noseka1/openshift-auto-upi/pull/22#discussion_r489138358.

hashnao committed:
Update openshift_libvirt.md

Moving the paragraph to the front page.

hashnao committed:
Update README.md

hashnao committed:
Update README.md

hashnao committed:
Update helper.ini

hashnao committed:
Update openshift_libvirt.md

hashnao committed:
Update delete_libvirt.yml

noseka1 commented:
@hashnao Thanks for the additional changes! Will merge this branch into master shortly.
